src/status_im/bots/models.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im.bots.models)
```

Comment:
The main reason for having models namespaces vs just defining the functions in events namespace is that, in some cases, the latter leads to circular dependencies, for example `status-im.chat.events` needs to require `status-im.chat.events.input` (for handlers to be registered), but `status-im.chat.events.input` needs some db "model" functions from `status-im.chat.events`.